src/frontend/src/redux/search/actions.ts

```diff
@@ -21,8 +22,47 @@ import {
   LOADING_EXPUNGEMENT_PACKET_COMPLETE,
 } from "./types";
 import { AliasData } from "../../components/RecordSearch/SearchPanel/types";
-import { RecordData } from "../../components/RecordSearch/Record/types";
+import {
+  RecordData,
+  ShortLabel,
+} from "../../components/RecordSearch/Record/types";
 import { updateStats } from "../statsSlice";
+import { getShortLabel } from "../../components/RecordSearch/Record/util";
+
+function isExludedCharge({ statute, level, name }: ChargeData) {
+  const chapter = Number(statute.slice(0, 3));
+  const lowerLevel = level.toLowerCase();
+
+  if (name.toLowerCase().includes("pedestrian j-walking")) return true;
+  if (level.toLowerCase().includes("infraction")) return true;
+
+  // 813 	Driving Under the Influence of Intoxicants
+  if (isNaN(chapter) || chapter === 813) return false;
+
+  if ((chapter >= 801 && chapter <= 826) || [481, 482, 483].includes(chapter)) {
+    if (!lowerLevel.includes("felony") && !lowerLevel.includes("misdemeanor")) {
+      return true;
```

Comment:
Where does this logic come from. Did you study up the statute yourself?

Reply:
For that, I tried to replicate the backend logic:
https://github.com/codeforpdx/recordexpungPDX/blob/3852214c7eb5ca9460ac24a608ac76aa7e26085f/src/backend/expungeservice/charge_classifier.py#L305-L328

with some clarifications from Michael:
<img width="663" alt="image" src="https://user-images.githubusercontent.com/120387970/220377326-df5479c0-b11d-44f6-855a-c710d2e691fd.png">

Reply:
Just as a general direction, I think it is nice to keep the frontend relatively free of legal logic if possible.